windows-graphics-drivers/script.sh

```diff
@@ -81,24 +81,42 @@ function expand_archives() {
   local path="${1}"
   local output_dir="$(mktemp --tmpdir=tmp -d)"
 
-  7zz -bso0 -bd -o"${output_dir}" x "${path}"
+  # Try unpacking the driver as a cabinet file. Note that we don't check the
+  # extension because they sometimes come as .exe, and it's not possibly to
+  # tell them apart from regular executables when they do.
+  cabextract -d "${output_dir}" "${path}"
+
+  if [ $? -ne 0 ]; then
```

Comment:
```suggestion
  if cabextract -d "${output_dir}" "${path}"; then
```

This isn't a big deal but I generally prefer this style because then you can ` set -e` and it won't fail out.

Reply:
Yeah, much better, and I intend to use `set -e` on these scripts eventually (unless they get rewritten into something more sensible first).